front/pages/api/w/[wId]/feature-flags.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+  WhitelistableFeature,
+  WithAPIErrorResponse,
+} from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { withSessionAuthenticationForWorkspace } from "@app/lib/api/wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { getFeatureFlags } from "@app/lib/auth";
+import { apiError } from "@app/logger/withlogging";
+
+export type GetWorkspaceFeatureFlagsResponseType = {
+  feature_flags: WhitelistableFeature[];
+};
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    WithAPIErrorResponse<GetWorkspaceFeatureFlagsResponseType>
+  >,
+  auth: Authenticator
+): Promise<void> {
+  const owner = auth.getNonNullableWorkspace();
+
+  switch (req.method) {
+    case "GET":
+      const feature_flags = await getFeatureFlags(owner.id);
+      return res.status(200).json({ feature_flags });
```

Comment:
`nit` on the private API we use camelCase.

Reply:
Yes you're right let's edit this 👍